macOS/DuckDuckGo/Common/Extensions/NSPopoverExtension.swift

```diff
@@ -75,13 +75,7 @@ extension NSPopover {
         var frame = frame
         let boundingFrame = self.boundingFrame
         if !boundingFrame.isInfinite, boundingFrame.width > frame.width {
-            var rightEdge = boundingFrame.maxX
-            if positioningView?.window?.isFullScreen == true {
-                // Move the popover to the left in full screen to ensure it's fully shown on screen.
-                // This fixes presenting from "More Options" menu.
-                rightEdge -= 48
```

Comment:
This was a bad idea after all. This change brings back the original implementation here.